packages/features/bookings/Booker/components/Header.tsx

```diff
@@ -42,7 +43,12 @@ export function Header({
     [setLayout, layout]
   );
 
-  if (isMobile || !enabledLayouts) return null;
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  useEffect(() => {
+    setIsEmbed(window?.isEmbed?.() ?? false);
+  });
+
+  if (isMobile || isEmbed || !enabledLayouts) return null;
```

Comment:
Why have we moved this check in here? We don't want to hide the entire header. Just the layout

Reply:
We will make this changes